File: dohyeon/Programmers/Level_0/Lv0_콜라츠_수열_만들기.py
def solution(n):
    answer = []
    answer.append(n)
    while True :
        if n % 2 == 0 :
            n = int(n / 2)
        else :
            n = 3 * n + 1
        answer.append(n)
        if n == 1 :
            break
    return answer

print(solution(10))

File: seongwon/Programmers/Level_0/Lv0_콜라츠_수열_만들기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(int n) {
    vector<int> answer;
    int i = 1;
    answer.push_back(n);
    while(n != 1){
        if(n%2==0){
            n /= 2;
            answer.push_back(n);
        }
        else{
            n = 3*n+1;
            answer.push_back(n);
        }
        i++;
    }
    return answer;
}